assets/js/newsletter.js

```diff
@@ -0,0 +1,15 @@
+---
+# Empty front matter to enable Jekyll processing
+---
+$("#newsletter-modal").on('shown.bs.modal', function(w, d, s, o, f, js, fjs) {
```

Comment:
I find this very difficult to read. Can you please use proper names of variables?

Reply:
As said elsewhere, this is a copy-paste snippet from the provider. I have no idea what the variables mean :-P. Though I should then probably state this fact in the comment.